client/src/main/java/org/apache/nemo/client/JobLauncher.java

```diff
@@ -122,6 +123,7 @@ public static void main(final String[] args) throws Exception {
    */
   public static void setup(final String[] args) throws InjectionException, ClassNotFoundException, IOException {
     // Get Job and Driver Confs
+    LOG.info("Project Root Path: {}", Util.fetchProjectRootPath());
```

Comment:
Does this always return the correct root path? Wouldn't it be better to set `NEMO_HOME` variable and fetch it?

Reply:
I'll make it so that it tries first for the NEMO_HOME system environment variable, and to fall back to the usual method if it is not provided. Thanks!